kie-spring/pom.xml

```diff
@@ -258,6 +258,7 @@
           <systemPropertyVariables>
             <kie.maven.settings.custom>${project.build.testOutputDirectory}/kie-custom-settings.xml</kie.maven.settings.custom>
           </systemPropertyVariables>
+          <forkMode>always</forkMode>
```

Comment:
@psiroky would it be possible to have a fork for just single test? I am bit afraid this might be too heavy...